src/components/displayQuote.js

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react';
+
+export default function DisplayQuote() {
+  const [data, setData] = useState({});
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    async function getQuote() {
+      try {
+        const response = await fetch('https://api.api-ninjas.com/v1/quotes?category=computers', {
+          method: 'GET',
+          headers: { 'X-Api-Key': 'a4jkLzxa11Fyl351KUe2lQ==OqEqrHKzRyti4wax' },
+          contentType: 'application/json',
+        });
+        const result = await response.json();
+        setData(result[0]);
+        setLoading(false);
+      } catch (e) {
+        setError(true);
+        setLoading(false);
+        throw new Error(e);
+      }
+    }
+    getQuote();
+  }, []);
```

Comment:
## Recommendation Changes 
Your work is nice, but let me mark this change as a recommendation which I highly recommend you work on 👍🏽 

- [ ] Good job on your implementation. Things are working well but it's highly considered to pass in dependencies to the dependency array of the useEffect hook. Any variable used in the hook that is not created inside it should be passed as a dependency array. Kindly add them 👍🏽

```javascript
 fetchQuote();
  }, [setData, setError, setLoading]);
```